bot/cogs/rss.py

```diff
@@ -0,0 +1,94 @@
+from asyncio import sleep
+from datetime import timedelta
+from typing import Dict, List
+
+import arrow
+import discord
+import feedparser
+
+from discord.ext.commands import Bot, Cog, Context, hybrid_command
+
+RPI_RSS_FEED_URL = "https://rpilocator.com/feed/"
+
+NOTIF_COOLDOWN = timedelta(hours=3)
+
+
+class NotifyMember:
+    def __init__(self, search: str, member: discord.Member, channel: discord.TextChannel):
+        self.search_term: str = search
+        self.member: discord.Member = member
+        self.notify_channel: discord.TextChannel = channel
+        self.notified: bool = False
+
+
+notifications: Dict[int, Dict[str, NotifyMember]] = {}
```

Comment:
Perhaps this should be within the `RSS` cog.

Reply:
That is probably preferred, but I think it is acceptable for now and I plan to refactor this cog fully in the near future so I will probably leave as-is.

Reply:
Very well.